exactonline/rawapi.py

```diff
@@ -163,3 +170,7 @@ def _set_tokens(self, jsondata):
         self.storage.set_access_expiry(int(time()) + expires_in)
         self.storage.set_access_token(decoded['access_token'])
         self.storage.set_refresh_token(decoded['refresh_token'])
+
+        if 'store_tokens' in dir(self.storage):
+            self.storage.store_tokens()
```

Comment:
Why would we need a separate function for this? Can't you override the set_access_token() and others to automatically save it immediately? That's what we do:

```
class AboutConfigStorage(ExactOnlineConfig):
...
    def set(self, section, option, value):
        key = self.to_key(section, option)
        aboutconfig(key, value, set=True)
```

Where `aboutconfig(..., set=True)` writes to the database immediately.

And if you feel that you need to write just once, just tack on a call to store_tokens() at an overloaded set_refresh_token().